core/nwb.behavior.yaml

```
@@ -38,6 +38,24 @@ groups:
         is 'meters'. Actual stored values are not necessarily stored in these units.
         To access the data in these units, multiply 'data' by 'conversion' and add 'offset'.
       required: false
+    - name: bounds
+      dtype: float32
```

`bounds` should be the same dtype as `data`, which is specified above as `numeric`
```suggestion
      dtype: numeric
```